packages/mobile/src/verify/VerificationInputScreen.tsx

```diff
@@ -206,6 +219,18 @@ class VerificationInputScreen extends React.Component<Props, State> {
     navigateHome()
   }
 
+  onPressResend = () => {
```

Comment:
Why are we showing an error message when `!isRevealAllowed` instead of hiding the button?

Reply:
Well, I guess Corey or Nitya can answer this better, but I think the idea is similar to why we do not disable send button, when there is not enough funds: user knows that such thing exists and also knows its limitations.

Reply:
Ok cool, just wanted to make sure this was discussed! No objection from me if this was the product/design decision.